code/__HELPERS/icons.dm

```diff
@@ -905,3 +939,31 @@ The _flatIcons list is a cache for generated icon files.
 		mask.BecomeAlphaMask() //Make all the black pixels vanish (fully transparent), leaving only the white pixels.
 		main.AddAlphaMask(mask) //Make the pixels in the main icon that are in the transparent zone of the mask icon also vanish (fully transparent).
 		return main
+
+//For creating consistent icons for human looking simple animals
+/proc/get_flat_human_icon(icon_id, datum/job/J, datum/preferences/prefs, dummy_key, showDirs = cardinal, outfit_override = null)
+	var/static/list/humanoid_icon_cache = list()
+	if(!icon_id || !humanoid_icon_cache[icon_id])
+		var/mob/living/carbon/human/dummy/body = generate_or_wait_for_human_dummy(dummy_key)
+
+		if(prefs)
+			prefs.copy_to(body)
+		if(J)
+			J.equip(body, TRUE, FALSE)
+		else if(outfit_override)
+			body.equipOutfit(outfit_override, visualsOnly = TRUE)

```

Comment:
@Aurorablade This is what I'm talking about, use the outfit_override argument to force it to be a defined tourist outfit-- perhaps that'll work better than the maketourist proc

Reply:
that still wouldn't help cuase i can't make it look like the actual mob right now... with my overlays its like this https://puu.sh/C0l9h/12e1423043.png

Reply:
Hahaha dang